documentation/memory/step2/using_heap_snapshot.md

```diff
@@ -6,4 +6,37 @@ can also compare multiple snapshots to see differences over time.
 
 ## How To
 
-//TODO
+### Get the Heap Snapshot
+
+The simplest way to get a Heap Snapshot is to connect a debugger to your process running locally and go to Profiling, choose to take a memory snapshot.
+
+If you need a snapshot form a working process, like an application running on a server, you should implement getting it using:
+
+```js
+require('v8').writeHeapSnapshot()
+```
+
+Check [writeHeapSnapshot docs](https://nodejs.org/dist/latest-v12.x/docs/api/v8.html#v8_v8_writeheapsnapshot_filename) for file name options
+
+You need to have a way to invoke it without stopping the process, so calling it in a http handler or as a reaction to a signal from the operating system is advised.  
```

Comment:
does it make sense to show example code here? for both http end point and the signal handler? while it will come handy for a good programmer, having it here makes it readily consumable?

Reply:
I wasn't sure if it belongs in this document. Can do if it's the desired content.
The exercise covers it, at least partially.